.pre-commit-config.yaml

```diff
@@ -667,20 +656,6 @@ repos:
           - requirements/static/ci/crypto.in
 
 
-      - id: pip-tools-compile
```

Comment:
If we are dropping python 3.5 support for windows and not linux we need to update the changelog and highlight it in the release notes for aluminium.

Reply:
Done